foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,54 @@
-Don't forget to commit early & often! You can [reference the Commit Message lesson here](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages)!
-
-1.  Start a new Git repo for your project.
-2.  Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file).  This game is going to be played completely from the console, so don't worry about putting anything else in there.
-3.  Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'.  We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-4.  Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
-    *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
-
-5.  **Important note:** you want to `return` the results of this function call, _not_ `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
-
-    ~~~javascript
-    function playRound(playerSelection, computerSelection) {
-      // your code here!
-    }
+Remember to commit early and often! To refresh your memory, check the [commit messages lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages).
+
+1. Create a new Git repository for your project.
+1. Create a blank HTML document with a script tag.
+   1. We recommend writing the code in an external JavaScript file and linking it to the HTML file. You play this game entirely on the console, so don't worry about writing anything else in the HTML file.
+1. Your game plays against the computer—create a function to `return` the computer's choice:
+   1. Create a new function called `getComputerChoice`.
+   1. Make `getComputerChoice` *randomly* `return` one of the following string values: `"Rock"`, `"Paper"` or `"Scissors"`.
+   1. Remember to test what your function returns with [browser's developer tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) or `console.log`.
+1. Your game plays turn by turn—create a function to take the human and computer choices, then `return` the round winner:
+   1. Create a new function called `playRound`.
+   1. Define two parameters on `playRound`: `playerSelection` and `computerSelection`. Use those two parameters to take the human and computer choices as arguments.
+   1. Make `playRound` `return` a string value representing the round winner, such as: `"You Lose! Paper beats Rock"`.
```

Comment:
```suggestion
   - Create a new function called `playRound`.
   - Define two parameters on `playRound`: `playerSelection` and `computerSelection`. Use those two parameters to pass the human and computer choices as arguments.
   - Make your function's `playerSelection` parameter case-insensitive (so users can input "rock", "ROCK", "RocK" or any other variation).
   - Write the code for your `playRound` function to `return` a string value representing the round winner, such as: "You Lose! Paper beats Rock".
   - example code:
```

- reworded some and added the case-insensitive part of the assignment back in.
- the below code should fit below the last list item, so indentation needs to increase. Feel free to reword the last list item to make it clear this is code to test if the function works.